setup.py

```diff
@@ -31,14 +31,14 @@
     zip_safe=False,
     python_requires=">3, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
     install_requires=[
-        "click>=7.1.1",
+        "click>=7.1.1, <8",
         "click_plugins>=1.1.1",
         "colorama>=0.4.3",
         "c42eventextractor==0.4.1",
         "keyring==18.0.1",
         "keyrings.alt==3.2.0",
         "pandas>=1.1.3",
-        "py42>=1.11.1",
+        "py42>=1.13.0",
```

Comment:
I also found that the current min py42 version is wrong, 1.13.0 added some methods that the latest CLI release uses

Reply:
woops nvm this was caused by my branch not being caught up